macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -14,9 +14,10 @@
     'disable_test_alerts': false,
     'disable_source_freshness_alerts': false,
     'disable_run_results': false,
+    'disable_autoupload_dbt_artifacts': false,
     'disable_skipped_model_alerts': true,
     'disable_skipped_test_alerts': true,
-    'dbt_artifacts_chunk_size': 50,
+    'insert_rows_batch_size': 5000,
```

Comment:
Let's keep it with the same name and value to keep backwards if someone defined it already?

Reply:
It's just that this makes _specifically_ the artifacts uploading aware of the insertion method.
I'd need to accept a `chunk_size` on `insert_rows()` even though most of the time it will not be used, and make `upload_artifacts_to_table()` pass that var.

It's not the worst, but I also don't like it that much.
What do you think?